pytest-embedded/pytest_embedded/plugin.py

```diff
@@ -988,6 +999,7 @@ def _fixture_classes_and_options(
                 kwargs[fixture] = {
                     'msg_queue': msg_queue,
                     'port': port,
+                    'port_location': port_location,
```

Comment:
also need to add this line here: https://github.com/espressif/pytest-embedded/pull/141/files#diff-b5a6b0a6e20cb69a3ce927b538961a62b3a189870fb359992536e7cc2618cfa5R966